### PR TITLE
Asset pipeline

### PR DESCRIPTION
**Pull request by techgique**
Please test by creating a new Rails app with this version of Orchid. Run the generator and then try adding both global and conditional scripts, stylesheets, and images. If you never see a precompilation error/warning, I'll be able to die in peace. :-D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,34 +57,93 @@ And then execute:
 bundle install
 ```
 
-## Usage
 
-Once orchid is installed successfully, run a generator to prepare your new rails app. Run this with a `--help` to see what will be changed before you begin. This will NOT erase or overwrite any configuration files, only example configuration files.
+## Usage
+Once Orchid is installed successfully, run the generator to prepare your new rails app. Run this with a `--help` to see which app files will be changed before you begin.
```
Comment (jduss4) [inline README.md]: I realize this was in before your PR, but the `--help` option suggests some arguments that we aren't respecting:

```
Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist
```

**Comment by techgique** [comment]
Okay, I definitely was in a hurry when I pushed and created my PR on Friday. 😒 

Made some basic fixes as well as a very important change to `application.js` for handling `global/` JS files.

**Comment by jduss4** [comment]
Awesome, it all works for me!

I'm happy to merge it, but first do you think it would be appropriate to add some of the comments that are output to the terminal in this home page view?  Then people could review them more readily as they start working on the app.

image

**Comment by techgique** [comment]
Yeah, I think that's a good idea! Do you think it's good to be in both places so one can read it before they start the app?

**Comment by jduss4** [comment]
Yeah, although it's kind of a pain for us to write it down in two places, I think that it would helpful to have it in both places, or at least a shoutout to look at the README or some centralized file?

**Comment by techgique** [comment]
Yeah duplicating / making more work is the drawback.
We could just have the generator and the index page display a link to the README on GitHub.
How's that sound?

**Comment by jduss4** [comment]
I think that sounds good!  We could mention "for information about updating templates, assets, and more" or something before the link to make it clear

**Comment by techgique** [comment]
Yes, yes. I'll work on it :)

**Comment by jduss4** [comment]
Thanks for moving everything into the readme!  I don't think I realized how involved that would end up being!!

**Pull request merged**